modules/minimap2/align/main.nf

```diff
@@ -4,16 +4,20 @@ process MINIMAP2_ALIGN {
 
     conda (params.enable_conda ? 'bioconda::minimap2=2.21' : null)
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/minimap2:2.21--h5bf99c6_0' :
-        'quay.io/biocontainers/minimap2:2.21--h5bf99c6_0' }"
+        'https://depot.galaxyproject.org/singularity/mulled-v2-66534bcbb7031a148b13e2ad42583020b9cd25c4:1679e915ddb9d6b4abda91880c4b48857d471bd8-0' :
+        'quay.io/biocontainers/mulled-v2-66534bcbb7031a148b13e2ad42583020b9cd25c4:1679e915ddb9d6b4abda91880c4b48857d471bd8-0' }"
 
     input:
     tuple val(meta), path(reads)
     path reference
+    val sam_format
+    val cigar_paf_format
+    val cigar_bam
 
     output:
-    tuple val(meta), path("*.paf"), emit: paf
-    path "versions.yml" , emit: versions
+    tuple val(meta), path("*.paf"), emit: paf, optional: true
+    tuple val(meta), path("*.sam"), emit: sam, optional: true
```

Comment:
```suggestion
    tuple val(meta), path("*.paf"), optional: true, emit: paf
    tuple val(meta), path("*.sam"), optional: true, emit: sam
```
Not important, but normally we put optional first (I think...)

Reply:
I changed it as suggested.